src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -200,6 +200,71 @@ void BLEManagerImpl::sl_ble_event_handling_task(void * args)
     }
 }
 
+uint32_t rsi_ble_add_matter_service(void)
+{
+    uuid_t custom_service                                   = { RSI_BLE_MATTER_CUSTOM_SERVICE_UUID };
+    custom_service.size                                     = RSI_BLE_MATTER_CUSTOM_SERVICE_SIZE;
+    custom_service.val.val16                                = RSI_BLE_MATTER_CUSTOM_SERVICE_VALUE_16;
+    uint8_t data[RSI_BLE_MATTER_CUSTOM_SERVICE_DATA_LENGTH] = { RSI_BLE_MATTER_CUSTOM_SERVICE_DATA };
+
+    static const uuid_t custom_characteristic_RX = {
+        .size     = RSI_BLE_CUSTOM_CHARACTERISTIC_RX_SIZE,
+        .reserved = { RSI_BLE_CUSTOM_CHARACTERISTIC_RX_RESERVED },
+        .val      = { .val128 = { .data1 = { RSI_BLE_CUSTOM_CHARACTERISTIC_RX_VALUE_128_DATA_1 },
+                                  .data2 = { RSI_BLE_CUSTOM_CHARACTERISTIC_RX_VALUE_128_DATA_2 },
+                                  .data3 = { RSI_BLE_CUSTOM_CHARACTERISTIC_RX_VALUE_128_DATA_3 },
+                                  .data4 = { RSI_BLE_CUSTOM_CHARACTERISTIC_RX_VALUE_128_DATA_4 } } }
+    };
+
+    rsi_ble_resp_add_serv_t new_serv_resp = { 0 };
+    rsi_ble_add_service(custom_service, &new_serv_resp);
+
+    // Adding custom characteristic declaration to the custom service
+    SilabsBleWrapper::rsi_ble_add_char_serv_att(
+        new_serv_resp.serv_handler, new_serv_resp.start_handle + RSI_BLE_CHARACTERISTIC_RX_ATTRIBUTE_HANDLE_LOCATION,
+        RSI_BLE_ATT_PROPERTY_WRITE | RSI_BLE_ATT_PROPERTY_READ, // Set read, write, write without response
+        new_serv_resp.start_handle + RSI_BLE_CHARACTERISTIC_RX_VALUE_HANDLE_LOCATION, custom_characteristic_RX);
+
+    // Adding characteristic value attribute to the service
+    SilabsBleWrapper::rsi_ble_add_char_val_att(
+        new_serv_resp.serv_handler, new_serv_resp.start_handle + RSI_BLE_CHARACTERISTIC_RX_VALUE_HANDLE_LOCATION,
+        custom_characteristic_RX,
+        RSI_BLE_ATT_PROPERTY_WRITE | RSI_BLE_ATT_PROPERTY_READ, // Set read, write, write without response
+        data, sizeof(data), ATT_REC_IN_HOST);
+
+    static const uuid_t custom_characteristic_TX = {
```

Comment:
same comment

Reply:
changed static const -> constexpr to the variable used in the function.